src/components/withFormField.tsx

```diff
@@ -113,11 +118,13 @@ export const withFormField = <OriginalProps extends {}>(Component: React.Compone
       />
       { label && <H5 className="label media-label">{ label }</H5> }
       { !isInvalid && hint && <H4 className="hint">{ hint }</H4> }
-      { form.errors[field.name] && form.touched[field.name] && (
-        <div className="error-wrapper">
-          <H5 className="error">{ t(form.errors[field.name]?.toString() || '') }</H5>
-        </div>
-      ) }
+      <div className="error-wrapper">
+        <H6 className="error">
+          {typeof form.errors[field.name] === 'string' && form.touched[field.name]
+            ? t(fieldError?.toString() || '')
+            : t(fieldError?.error || '', { length: fieldError?.param })}
```

Comment:
bad. what is the length? moreover, you set the only value in param. what about the case if we need more than 1 param? you have to fix it